### ISSUE TITLE
TypeError: Object of type 'PosixPath' is not JSON serializable

### ISSUE THREAD

**Issue by jdkent** [neuroscout/neuroscout-cli#110]
Command I used:
```
docker run --rm -it -v /home/jdkent/projects/daBombDotCom:/out neuroscout/neuroscout-cli run /out
AY4yO
```

output (including traceback):
```
INFO:root:Analysis ID : AY4yO
INFO:root:Downloading bundle...
INFO:root:Bundle installed at /work/Life/neuroscout-bundles/AY4yO
[INFO   ] Cloning https://github.com/neuroscout-datasets/life-local to '/work/Life/preproc'
INFO   :datalad.distribution.clone:Cloning https://github.com/neuroscout-datasets/life-local to '/work/Life/preproc'
[PosixPath('Life/preproc/fmriprep/dataset_description.json')]
/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/bids/layout/validation.py:46: UserWarning: The ability to pass arguments to BIDSLayout that control indexing is likely to be removed in future; possibly as early as PyBIDS 0.14. This includes the `config_filename`, `ignore`, `force_index`, and `index_metadata` arguments. The recommended usage pattern is to initialize a new BIDSLayoutIndexer with these arguments, and pass it to the BIDSLayout via the `indexer` argument.
  warnings.warn("The ability to pass arguments to BIDSLayout that control "
Traceback (most recent call last):
  File "/opt/miniconda-latest/envs/neuro/bin/neuroscout", line 33, in <module>
    sys.exit(load_entry_point('neuroscout-cli', 'console_scripts', 'neuroscout')())
  File "/src/neuroscout/neuroscout_cli/cli.py", line 65, in main
    command(deepcopy(args)).run()
  File "/src/neuroscout/neuroscout_cli/commands/run.py", line 21, in run
    bundle_path = install.run(download_data=(not upload_only))
  File "/src/neuroscout/neuroscout_cli/commands/install.py", line 117, in run
    return self.download_data()
  File "/src/neuroscout/neuroscout_cli/commands/install.py", line 92, in download_data
    raise(e)
  File "/src/neuroscout/neuroscout_cli/commands/install.py", line 74, in download_data
    derivatives=self.preproc_dir, index_metadata=False)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/bids/layout/layout.py", line 151, in __init__
    database_path, reset_database, config, init_args)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/bids/layout/db.py", line 48, in __init__
    self.reset_database(init_args, config)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/bids/layout/db.py", line 108, in reset_database
    layout_info = LayoutInfo(**init_args)
  File "<string>", line 4, in __init__
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/sqlalchemy/orm/state.py", line 433, in _initialize_instance
    manager.dispatch.init_failure(self, args, kwargs)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/sqlalchemy/orm/state.py", line 430, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/bids/layout/models.py", line 48, in __init__
    json_data = json.dumps(init_args[col])
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'PosixPath' is not JSON serializable
```

**Comment by effigies** [comment]
Needs fixing in https://github.com/bids-standard/pybids/issues/655.

**Comment by adelavega** [comment]
Thanks! @jdkent you should also be able to rollback neuroscout-cli docker image to this tag: `version-0.3.7` see if that helps

**Comment by adelavega** [comment]
Closed by #112
